book/website/reproducible-research/licensing/licensing-ethical-source.md

```diff
@@ -0,0 +1,51 @@
+(rr-licensing-ethical)=
+# 'Ethical Source'
+
+The ethical source movement seeks to affirmatively protect specific user rights by curtailing freedom 0, the freedom to use software 'for any purpose' and prohibiting the use of the software for unethical purposes.
+Consequently, these licence by the classical definitions of free and open-source software from the FSF and OSI would not be considered free or open-source licences, they do however generally resemble them in the  other three criteria of the definition.
+Their merits versus conventional open-source licenses has been the subject of some debate, and their adoption has thus far been relatively limited.
+
+A leading figure in this movement is Coraline Ada Ehmke creator of the [Contributor Covenant](https://www.contributor-covenant.org/), the [Hippocratic License](https://firstdonoharm.dev/), and the founder in 2020 of the [Organisation for Ethical Source](https://ethicalsource.dev/).
+In the words of its advocates, [Ethical Source](https://ethicalsource.dev/) was created to "empower developers, giving us the freedom and agency to ensure that our work is being used for social good and in service of human rights."
+Motivated by the growing use of open source software for technologies such as mass surveillance and racial profiling, the movement aims to reduce this "misuse" of open-source software.
+
+Ethical Source extends upon the principles of open source to provide developers additional means to ensure their work is used for applications aligned with ethical values important to them like:
+
+- Advocating for workers' rights and human rights
+- Ensuring protections against violence and discrimination
+- Protecting privacy
+
+A list of Ethical Licenses and their targeted applications can be found on the [Ethical Source website](https://ethicalsource.dev/licenses/).
+The core Hippocratic licence, for example, prohibits a variety of human rights abuses and mandates equal pay for equal work.
+It can be further extended using [their licence building tool](https://firstdonoharm.dev/build/) with modules covering:
```

Comment:
Is this overview of the models complete? Otherwise maybe select a couple instead of the whole list?

Reply:
Yes I believe that is a complete list of their modules. Do you think I should just pick a few examples instead of the full list?